exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/SequencesTests.kt

```diff
@@ -132,6 +133,21 @@ class SequencesTests : DatabaseTestsBase() {
         }
     }
 
+    @Test
+    fun testSequenceExists() {
+        withDb {
+            if (currentDialectTest.supportsCreateSequence) {
+                try {
+                    SchemaUtils.createSequence(myseq)
+
+                    assertTrue(myseq.exists())
+                } finally {
+                    SchemaUtils.dropSequence(myseq)
+                }
+            }
+        }
+    }
```

Comment:
`existingSequences()` will also be accessible from the public API via the current dialect, so it'd be good to test too.

It'd be great to test that both new functions cover manually created sequences (as done above), as well as autogenerated sequences. Both when an explicit sequence name is provided to `autoIncrement()` and when one isn't (for Oracle, that the existing result matches the result of `autoincSeq`).